chain/bitcoin/bitcoin.go

```diff
@@ -109,6 +109,16 @@ func NewClient(opts ClientOptions) Client {
 	}
 }
 
+// LatestBlock returns the height of the longest blockchain.
+func (client *client) LatestBlock(ctx context.Context) (pack.U64, error) {
+	var resp int64
+	if err := client.send(ctx, &resp, "getblockcount"); err != nil {
+		return pack.NewU64(0), fmt.Errorf("get block count: %v", err)
+	}
+
```

Comment:
Need to bounds check the int64 to make sure that it is not negative before casting it to a U64. If it is negative, return an error.